docs/process/release.md

```diff
@@ -122,9 +122,20 @@ As commits are cherry-picked when PRs are merged, creating the release should be
 ```bash
 git commit -a -m 'release: vXX'
 git tag 'vXX'
+```
+
+The package versions we are about to publish are derived from the git tag that
+we just created. Double check that the versions are correct by running the
+following command.
+
+```bash
+yarn admin packages --version
+```
 
-# Make sure to run these commands together, as missing tags can cause CI
-# failures.
+Now push the commit and the tag to the upstream repository.
+# Make sure to run these commands together, as missing tags can cause CI failures.
```

Comment:
Nit: This should probably be bolded rather than a header, not sure why it was a header already.

Reply:
changed to bold